src/NzbDrone.Core/Notifications/MediaBrowser/MediaBrowserSettings.cs

```diff
@@ -12,38 +13,37 @@ public MediaBrowserSettingsValidator()
         {
             RuleFor(c => c.Host).ValidHost();
             RuleFor(c => c.ApiKey).NotEmpty();
+            RuleFor(c => c.UrlBase).ActuallyValidUrlBase();
         }
     }
 
     public class MediaBrowserSettings : IProviderConfig
     {
         private static readonly MediaBrowserSettingsValidator Validator = new MediaBrowserSettingsValidator();
 
-        public MediaBrowserSettings()
-        {
-            Port = 8096;
-        }
-
         [FieldDefinition(0, Label = "Host")]
         public string Host { get; set; }
 
         [FieldDefinition(1, Label = "Port")]
-        public int Port { get; set; }
+        public int Port { get; set; } = 8096;
 
         [FieldDefinition(2, Label = "Use SSL", Type = FieldType.Checkbox, HelpText = "Connect to Emby over HTTPS instead of HTTP")]
         public bool UseSsl { get; set; }
 
-        [FieldDefinition(3, Label = "API Key", Privacy = PrivacyLevel.ApiKey)]
+        [FieldDefinition(3, Label = "Url Base", Type = FieldType.Textbox, Advanced = true, HelpText = "Adds a prefix to the Emby server url, see http://[host]:[port]/[urlBase]")]
```

Comment:
`see x` would refer to a link, instead use the same wording we use for SABnzbd: `Adds a prefix to the Sabnzbd url, e.g. http://[host]:[port]/[urlBase]/api`

Reply:
Should I fix the "see x" phrase in other mentions as well? I've used that one, because Deluge actually uses it. ( And without shame, I copy pasted it. )

Reply:
Ahh, yeah, it's be good to fix that one too